e2e/cypress/integration/account_settings/display/account_settings_position_spec.js

```diff
@@ -55,35 +53,32 @@ describe('Account Settings > General > Position', () => {
     it('MM-T2064 Position / 128 characters', () => {
         const longPosition = 'Master Hacker II'.repeat(8);
 
-        // # Open the hamburger menu
-        cy.findByLabelText('main menu').should('be.visible').click();
-
-        // # Click on Account settings menu item
-        cy.findByText('Account Settings').should('be.visible').click();
-
-        // # Fill-in the position field with a value of 128 characters
-        cy.findByText('Position').should('be.visible').click();
-        cy.get('#position').type(longPosition);
-        cy.get('#saveSetting').click();
-        cy.get('#position').should('not.be.visible');
-
-        cy.findByText('Position').should('be.visible').click();
-        cy.get('#position').invoke('val').then((val) => {
-            // # Verify that the input value is 128 characters
-            expect(val.length).to.equal(128);
+        // # Open 'Account Settings' modal and view the default 'General Settings'
+        cy.uiOpenAccountSettingsModal().within(() => {
+            const minPositionHeader = () => cy.findByRole('heading', {name: 'Position'});
+            const maxPositionInput = () => cy.findByRole('textbox', {name: 'Position'});
+
+            // # Fill-in the position field with a value of 128 characters
+            minPositionHeader().click();
+            maxPositionInput().type(longPosition);
+            cy.uiSave();
+            maxPositionInput().should('not.exist');
+
+            minPositionHeader().click();
+            maxPositionInput().invoke('val').then((val) => {
+                // # Verify that the input value is 128 characters
```

Comment:
```suggestion
                // * Verify that the input value is 128 characters
```

Reply:
Done update. I didn't notice since it's from the original script.